### DISCUSSION TITLE
SourceCodePro ligature incorrectly printed when whitespace rendering is enabled

### DISCUSSION THREAD

**Discussion by m-torhan** [helix-editor/helix#10960] [Troubleshooting] [answered]
### Summary

If whitespace rendering is enabled, then typing "l l" results in "l·l" ligature being rendered.

### Reproduction Steps

 

I tried this:

1. Enable whitespace rendering
```toml
[editor.whitespace.render]
space = "all"
```
2. Run `helix` and type "l l".

I expected this to happen:
![image](https://github.com/helix-editor/helix/assets/38363382/d5854a15-a74c-4365-9bec-9f3b40a42e0a)

Instead, this happened:
![image](https://github.com/helix-editor/helix/assets/38363382/3fdce116-67af-47fa-8168-9f21dce5f235)

### Platform

Linux

### Terminal Emulator

kitty 0.33.1

### Installation Method

pacman

### Helix Version

helix 24.3

**Comment by the-mikedavis** [comment, marked as answer]
We don't control how the terminal creates or renders ligatures. I expect that if you run `echo "l·l"` in your shell you'll see the same rendering

**Reply by m-torhan** [reply]
Yeah, you are right. Even typing it produces the ligature. I guess I will just change the character for whitespace rendering to solve that.
